### DISCUSSION TITLE
Adding my own CT data to the demos

### DISCUSSION THREAD

**Discussion by p2a3queraro** [CERN/TIGRE#395] [Q&A]
Hey, I ran the demos and managed to save some gifs, but I'm not sure about how to add my own CT data? Particularly interested in d08-algorithm03 and 02. 

Do I need just the CT data (no of projections, geometry etc) or a .vol/.raw/etc file?

**Comment by AnderBiguri** [comment]
Hi @p2a3queraro , there is a demo for your own data, demo 21. Have you looked at it?

**Reply by p2a3queraro** [reply]
runfile('//filestore.soton.ac.uk/users/pd3g21/mydesktop/TIGRE-master/TIGRE-master/Python/demos/try_demo.py', wdir='//filestore.soton.ac.uk/users/pd3g21/mydesktop/TIGRE-master/TIGRE-master/Python/demos')
Traceback (most recent call last):
  File \\filestore.soton.ac.uk\users\pd3g21\mydesktop\TIGRE-master\TIGRE-master\Python\demos\try_demo.py:19 in <module>
    proj,geo, angles  = tigreio.NikonDataLoader(datafolder)
  File ~\AppData\Roaming\Python\Python39\site-packages\pytigre-2.2.0-py3.9-win-amd64.egg\tigre\utilities\io\NikonDataLoader.py:36 in NikonDataLoader
    folder, geometry, angles = readXtekctGeometry(filepath)
  File ~\AppData\Roaming\Python\Python39\site-packages\pytigre-2.2.0-py3.9-win-amd64.egg\tigre\utilities\io\NikonDataLoader.py:91 in readXtekctGeometry
    geometry.COR = -float(cfg["CentreOfRotationTop"])
  File C:\ProgramData\Anaconda3\lib\configparser.py:1257 in __getitem__
    raise KeyError(key)
KeyError: 'CentreOfRotationTop'

yes, but I do get this ^

**Reply by AnderBiguri** [reply]
Is it a Nikon dataset? 

If yes, can you share with me the xteckt/geo file? Apparently they change their parameters in each version, so your (hidden) version may not be supported. Can change that easily. 

If not, then you are using the wrong function, of course.
